docs/administer/rbac.md

```diff
@@ -0,0 +1,435 @@
+
+## How to enable RBAC
+
+To enable or disable RBAC in Percona Everest, you can use a configuration flag that allows switching between RBAC-enabled and RBAC-disabled modes. By default, RBAC is disabled.
+
+Here's how you can enable RBAC:
+
+```sh
+apiVersion: v1
+data:
+  enabled: "false"
+  policy.csv: |
+    g, admin, role:admin
+kind: ConfigMap
+metadata:
+  name: everest-rbac
+  namespace: everest-system
+```
+
+## Policy definition in RBAC
+
+RBAC policies are the rules and guidelines that define how roles, permissions, and users are managed within RBAC. These policies ensure that users have appropriate access to resources based on their roles within Percona Everest.
+
+
+The policy definition in Percona Everest is:
+
+
+```sh
+p, <subject>, <resource-type>, <action>, <resource-name>
+```
+
+Where:
+
+**subject**: Refers to the name of the role or user. For example, `adminrole:role` or `admin`
+
+**resource-type**: Refers to the type of Everest resource, such as `namespaces`, `database-clusters`, `database-engines`, etc.
+
+**[action](#rbac-resources-and-permissions)**: Refers to the action the subject can perform. For example, `read`, `update`, `delete`, `create`, or `*`
+
+For in-depth information on the actions that a resource can perform, see the [resources and permissions](#rbac-resources-and-permissions) section.
+
+**resource-name**: Refers to a specific instance of the given resource-type. The argument should be prefixed with the namespace in which the resource is present. For example, `my-namespace/my-cluster-1`, `my-namespace-2/my-backup-1`, etc. You may also use a wildcard, such as `*`, `*/*`, or `my-namespace/*`
+
+### Admin policy
+
+Check out the policy for an **Admin** role:
+
+```sh
+    p, adminrole:role, namespaces, *, *
+    p, adminrole:role, database-engines, *, */*
+    p, adminrole:role, database-clusters, *, */*
+    p, adminrole:role, database-cluster-backups, *, */*
+    p, adminrole:role, database-cluster-restores, *, */*
+    p, adminrole:role, database-cluster-credentials, *, */*
+    p, adminrole:role, backup-storages, *, */*
+    p, adminrole:role, monitoring-instances, *, */*
+```
+
+### Read only policy
+
+Check out the policy for a **readonly** role:
+
+```sh
+    p, readonly:role, namespaces, read, *
+    p, readonly:role, database-engines, read, */*
+    p, readonly:role, database-clusters, read, */*
+    p, adminrole:role, database-cluster-backups, *, */*
+    p, adminrole:role, database-cluster-restores, *, */*
+    p, adminrole:role, database-cluster-credentials, read, */*
+    p, readonly:role, backup-storages, read, */*
+    p, readonly:role, monitoring-instances, read, */*
```

Comment:
Let's start with a very simple example instead, we have examples for detailed full roles below, here we just want users to understand the basic format.
```suggestion
### Example policies

The following policies grant `example-user` unrestricted access to database clusters and `read` access to database cluster backups, meaning that the user won't be able to `create` or `delete` any backup.

```sh
    p, example-user, database-clusters, *, */*
    p, example-user, database-cluster-backups, read, */*
```